bower.json

```diff
@@ -9,7 +9,7 @@
     "ember-qunit": "0.4.9",
     "ember-qunit-notifications": "0.0.7",
     "ember-resolver": "~0.1.20",
-    "jquery": "~1.11.3",
+    "jquery": "<=2.1.4",
```

Comment:
1. `materialize` requires al least `jquery@2.1.1` https://github.com/Dogfalo/materialize/blob/master/bower.json#L57
3. However there is an issue related to `materialize`'s datepicker with `jquery@2.2.0` (https://github.com/Dogfalo/materialize/issues/2883) which is solved only on `materialize@>=0.97.6`
3. `ember-cli-materialize` only supports `materializecss@=<0.97.5` because of  `"mdi-"` icons class removal (https://github.com/Dogfalo/materialize/commit/6ccc79bc6c4899d0eb25bbcdfd8add22f6652835)
4. Thus, we should require that `jquery` won't be higher than 2.1.4, which is the highest version of `jquery@2.1.x`